host-configs/lc-builds/toss3/icpx_X.cmake

```diff
@@ -5,12 +5,8 @@
 # SPDX-License-Identifier: (BSD-3-Clause)
 ###############################################################################
 
-set(CMAKE_CXX_FLAGS_RELEASE "${COMMON_FLAGS} -O3 -march=native -finline-functions" CACHE STRING "")
+set(CMAKE_CXX_FLAGS_RELEASE "${COMMON_FLAGS} -O3 -march=native -finline-functions --gcc-toolchain=/usr/tce/packages/gcc/gcc-8.1.0" CACHE STRING "")
```

Comment:
Should we be setting CMAKE_CXX_FLAGS so gcc will be added regardless of build type?

Reply:
Oops. You are correct, sir. I will push a change shortly.

Reply:
Change pushed.